### PR TITLE
Move legacy UI to its new home in preparation for Nimbus UI work

### PR DESCRIPTION
**Pull request by jodyheavener**
## This PR:

- Renames `app/experimenter/static/` to `app/experimenter/legacy-ui/`
- Moves `app/experimenter/templates/` into `app/experimenter/legacy-ui/`
- Updates references to the locations of these two mass-movements
- Adds a `.git-blame-ignore-revs` and references the mass-movement commit
- Adds a README in the `legacy-ui` directory to document the structure change
- Updates the root README to talk about the structure change and reference the `legacy-ui` README.

## Notes

You can learn more about why ignoring mass-refactoring commits is good and what it involves [here](https://akrabat.com/ignoring-revisions-with-git-blame/), but the key take-away is that you need to tell your local Git repo to observe this file:

```
git config blame.ignoreRevsFile .git-blame-ignore-revs
```

I have updated the root README to note this as well.

### CODE REVIEW

**Comment by jodyheavener** [comment]
To keep my sanity I'm going to hold off rebasing until the majority of reviews are in for this.

**Comment by jodyheavener** [comment]
I just want to say that I am pretty impressed with myself that this passed first try 😏

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# rename /static to /legacy-ui, move /templates 23-09-2020
+330dc37b30f63af505db6a843356d69d8af21bbc
```
Comment (jaredlockhart) [inline .git-blame-ignore-revs:2]: What does this do?

Comment (jaredlockhart) [reply .git-blame-ignore-revs:2]: Oh I see explained below

File: README.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 Done!
 
+Pro-tip: we have had at least one large code refactor. You can ignore specific large commits when blaming by setting the Git config's `ignoreRevsFile` to `.git-blame-ignore-revs`:
+
+```
+git config blame.ignoreRevsFile .git-blame-ignore-revs
+```
+
```
Comment (jaredlockhart) [inline README.md:68]: Neat! I didn't know you could do that

**Review by jaredlockhart — APPROVED** [review]
Awesome great job, tried locally everything looks correct, thnx @jodyheavener ! 🎉

**Pull request merged**